guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc

```diff
@@ -34,11 +39,13 @@ To enable the Secure Sockets Layer (SSL) protocol for these external databases,
 ----
 --foreman-db-sslmode verify-full
 --foreman-db-root-cert <path_to_CA>
+ifdef::katello,orcharhino,satellite[]
 --katello-candlepin-db-ssl true
 --katello-candlepin-db-ssl-verify true
 --katello-candlepin-db-ssl-ca <path_to_CA>
 --foreman-proxy-content-pulpcore-postgresql-ssl true
 --foreman-proxy-content-pulpcore-postgresql-ssl-root-ca <path_to_CA>
+endif::[]
 ----
 
 
```

Comment:
```suggestion
```

trailing whitespace?

Reply:
Was there before, but I can clean it up while I'm at it.